static/app/views/starfish/views/spanSummaryPage/sampleList/index.tsx

```diff
@@ -32,10 +32,12 @@ export function SampleList({groupId, transactionName, transactionMethod}: Props)
   const [highlightedSpanId, setHighlightedSpanId] = useState<string | undefined>(
     undefined
   );
-  const detailKey =
-    groupId && transactionName && transactionMethod
-      ? `${groupId}:${transactionName}:${transactionMethod}`
-      : undefined;
+
+  // A a transaction name is required to show the panel, but a transaction
```

Comment:
Should the transactionMethod Prop be optional too so it's explict?

Reply:
Yes, good point! Done